docs/new-version-branch.md

```diff
@@ -1,64 +1,101 @@
+1.  In `antora.yml`, in section `asciidoc.attributes`, DO NOT adjust relevant `-version` keys. They are required for local building.
+1.  In `site.yml`, in section `asciidoc.attributes`, DO NOT adjust relevant `-version` keys. They are used for local building and will be correctly set in the docs repo when doing a full build. NOTE: any attribute values defined here overwrite any attributes included via the `load-global-site-attributes.js` extension. 
+1.  Run a build by entering `npm run antora-local`. No build errors should occur.
+1.  Commit the changes and push the new `x.y` branch. This makes the branch available for futher processing. DO NOT CREATE A PR!
 
-20. In `site.yml` of the [docs](https://github.com/owncloud/docs/blob/master/site.yml) repo, adjust all `-version` keys in section `attributes` related to this repo according the new and former releases. Note that the values MUST NOT contain the trailing `@`. (The trailing @ character allows the value to be overwritten like from the corresponding `antora.yml` file which is only necessary for local building the corresponding docs-xxx repo.) Note that merging that PR should be _close before_ publishing the relevant code release.
+**Step 3: Protect the new branch**
 
-**Step 5: Set URI `latest` Path Part to 10.x**
+The branch has been pushed, add it to the rule set for protected version branches.
 
-21. Nothing needs to be done there. The moment when the new server release gets tagged - which is part of the release process - `latest` will be automatically set to the tagged release number. This works automatically up to version `10.20`. After that, backend admins need to be informed to updated the underlying process.
+1. Go to the github settings of this repository and change the protection of the branch list so that
+    the `x.y` branch just pushed before now gets included in the branch protection like the other active branches do.
 
-**Text Suggestion for Step 2**
+**Step 4: Configure the master branch to use the new `x.y` branch**
+
+This step is necessary to update the building process for content changes in this repo. 
+
+1. Create a new `changes_necessary_for_x.y` branch based on latest `origin/master`.
+1. In `.drone.star` set `latest_version` to `x.y` (on top in section `def main(ctx)`).
+1. In `antora.yml`, check if the `version:` key is set to `next`.
+1. In `site.yml` and in `antora.yml`, DO NOT adjust relevant `-version` keys.
+1. Run a build by entering `npm run antora-local`. No build errors or warnings should occur.
+1. Commit changes and push them. (Check the branch protection rules upfront so that the push passes.)
+1. Create a Pull Request `Changes necessary for x.y`, see the [text suggestion](#text-suggestion-for-step-4) below. When CI is green, all is done correctly, merge the PR when approved. This merge does NOT add the version to the main building process. You can now add at any time content changes to this version.
```

Comment:
Is it expected that they all start with `1`?

Reply:
Yes, sadly :sweat_smile: 
This is one of the downside of markdown where automatic numbering become a challenge.
But if you start a numbering always with e.g. `1.`, it will number it correctly :smiley: 

Asciidoc handles this differently and much more natural than MD

Reply:
again what learned